Sources/SwiftSyntax/generated/SyntaxTraits.swift

```diff
@@ -137,34 +137,17 @@ public protocol EffectSpecifiersSyntax: SyntaxProtocol {
     set
   }
   
-  var unexpectedBetweenAsyncSpecifierAndThrowsSpecifier: UnexpectedNodesSyntax? {
+  var unexpectedBetweenAsyncSpecifierAndThrowsClause: UnexpectedNodesSyntax? {
```

Comment:
We should also provide compatibility layers for `unexpectedBetweenAsyncSpecifierAndThrowsSpecifier` and `unexpectedAfterThrownError`.